CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (server) [#17094](https://github.com/cosmos/cosmos-sdk/pull/17094) Add duration `shutdown-grace` for resource clean up (closing database handles) before exit.
 * (x/auth/vesting) [#17810](https://github.com/cosmos/cosmos-sdk/pull/17810) Add the ability to specify a start time for continuous vesting accounts.
 * (baseapp) [#18071](https://github.com/cosmos/cosmos-sdk/pull/18071) Add hybrid handlers to `MsgServiceRouter`.
+* (x/auth) [#18281](https://github.com/cosmos/cosmos-sdk/pull/18281) Allow broadcast multi txs.
```

Comment:
nit, can you move the changelog to x/auth/CHANGELOG.md?

Reply:
auth haven't have a changelog yet so I created one.

Reply:
I think it does: https://github.com/cosmos/cosmos-sdk/blob/main/x/auth/CHANGELOG.md. Maybe rebase?